README.rst

```diff
@@ -319,23 +311,20 @@ The following procedure illustrates how to compile all the *openwisp-config* var
     # any arch/target is fine because the package is architecture indipendent
     arch="ar71xx"
     echo "CONFIG_TARGET_$arch=y" > .config;
-    echo "CONFIG_PACKAGE_openwisp-config-openssl=y" >> .config
-    echo "CONFIG_PACKAGE_openwisp-config-mbedtls=y" >> .config
-    echo "CONFIG_PACKAGE_openwisp-config-wolfssl=y" >> .config
-    echo "CONFIG_PACKAGE_openwisp-config-nossl=y" >> .config
+    echo "CONFIG_PACKAGE_openwisp-config=y" >> .config
     make defconfig
     make tools/install
     make toolchain/install
     make package/openwisp-config/compile
 
 Alternatively, you can configure your build interactively with ``make menuconfig``, in this case
-you will need to select the *openwisp-config* variant by going to ``Administration > openwisp``:
+you will need to select the *openwisp-config* by going to ``Administration > openwisp``:
```

Comment:
Better:

`you will need to select *openwisp-config* by going to ``Administration > openwisp``:`